Resources/doc/annotations/converters.rst

```diff
@@ -203,6 +203,33 @@ is accepted. You can be stricter with input given through the options::
     {
     }
 
+CurrentUser Converter
+~~~~~~~~~~~~~~~~~~~~~
+
+Converter Name: ``current_user``
+
+This converter injects the current user from the ``SecurityContext``, if available.
+
+This has several benefits:
+
+- Short syntax, reduce logic in controller
+- Alternative to ``getUser`` from base controller class, avoiding reliance on the
+  service container
+
+Its usage is simple. Provide the converter name and the parameter name you wish
+to be converted. These are both required in order to not conflict with the Doctrine
+ParamConverter.
+
+The example below illustrates applying this to one parameter, where the ``$recipient``
+could be handled by another ParamConverter.
+
+    /**
+     * @ParamConverter(converter="current_user", name="user")
+     */
+     public function sendMessageAction(User $currentUser, User $recipient)
```

Comment:
@adamquaile I saw that after reading the new comment down below. I wanted to comment here though, as I think that this addition would still be useful today.

@iltar yeah, that is right. Nevertheless the session user is often times the user entity as this is really comfortable and provides practically no downsides (except of this example right here of course :wink:). Or does it have any other negative impact?